src/components/data-display/MainDataDisplay.svelte

```diff
@@ -123,11 +128,14 @@
     const adjustForImpossible = (ammoData: AmmoToTData): void => {
         for (const [key, value] of Object.entries(ammoData)) {
             if (value >= 0) {
```

Comment:
Could we unnest these? We should be able to resolve this all at the same level

Reply:
Fixed